addons/project/views/project_task_views.xml

```diff
@@ -720,11 +720,11 @@
                           scales="month,year">
                     <field name="allow_milestones" invisible="1" />
                     <field name="is_private" invisible="1"/>
+                    <field name="priority" invisible="1"/>
```

Comment:
t possible to use the widget? That is the kind of things you want to explain in your commit message :angel: